lms/templates/courseware/progress.html

```diff
@@ -78,6 +78,26 @@ <h4 class="hd hd-4 title">${certificate_data.title}</h4>
                 </div>
                 %endif
 
+                %if enrollment_mode == CourseMode.AUDIT and upgrade_link:
```

Comment:
Could/should this be a shared block?  It's not necessary as part of this PR, unless you want to more generically sprinkle this around.

I imagine a future enhancement would be to not show this, or show something different, once it becomes impossible to pass?